gallery/screenshots.md

```diff
@@ -0,0 +1,18 @@
+# Screenshots
```

Comment:
Should we link to this markdown file from the main README?

Reply:
I don't think it's necessary. It's more of a place to point people to when they ask us for screenshots